.github/workflows/close-incomplete-inactive.yml

```diff
@@ -0,0 +1,17 @@
+name: Close incomplete inactive
+on:
+  schedule:
+    - cron: "0 0 * * *"
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Need more information
+        uses: actions-cool/issues-helper@v2
+        with:
+          actions: 'close-issues'
+          labels: 'status: incomplete'
+          inactive-day: 7
```

Comment:
We say 14 days in https://www.notion.so/mui-org/GitHub-issues-PRs-c1d7072e0c2545b0beb43b115f6030f6.

Reply:
Either way works. Maybe we can have it as 2 weeks?

Reply:
I have left a comment on Notion, to propose a change. AFAIK, we used 14 days because Sebastian was arguing that 30 days was fine.